pom.xml

```diff
@@ -76,6 +76,14 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        
+
+        <dependency>
+            <groupId>org.instancio</groupId>
+            <artifactId>instancio-core</artifactId>
+            <version>5.0.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
add documentation on our actual implementation

Reply:
Created a chore #83 for this so Jeffrey is unblocked for refactoring.